README.md

```diff
@@ -1,6 +1,6 @@
 # EN10168-schemas
 
-[![EN10168 Schema CI](https://github.com/thematerials-network/EN10168-schemas/actions/workflows/ci.yml/badge.svg)](https://github.com/thematerials-network/EN10168-schemas/actions/workflows/ci.yml)
+[![EN10168 Schema CI](https://github.com/material-identity/EN10168-schemas/actions/workflows/ci.yml/badge.svg)](https://github.com/material-identity/EN10168-schemas/actions/workflows/ci.yml)
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fs1seven%2FEN10168-schemas.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fs1seven%2FEN10168-schemas?ref=badge_shield)
```

Comment:
the FOSSA badge needs to be updated (or removed). It seems it dates from when the repo was still under s1seven org.

Reply:
Do you mean the second one?

`[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fs1seven%2FEN10168-schemas.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fs1seven%2FEN10168-schemas?ref=badge_shield)`

Reply:
yes